app/assets/stylesheets/app/tables.css.scss

```diff
@@ -1,3 +1,7 @@
 tr.row-with-actions {
   @extend tr.row-with-avatar;
 }
+
+td.icon, th.icon {
```

Comment:
Avoid qualifying class selectors with an element.<br>Each selector in a comma sequence should be on its own line